<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CancelOrderWindow</class>
 <widget class="QDialog" name="CancelOrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1561</width>
    <height>1072</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 28px;
    color:black;
	background-color: transparent;

}

QPushButton {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 28px;
    color: white;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                  stop:0 #FDC245, stop:1 #FB9E23);
border-radius: 10px;

}
QPushButton:hover {
background-color:#FAC85C;
}

QLineEdit{
font-size: 24px;
border-radius: 10px;
}</string>
  </property>
  <widget class="QLabel" name="label_question">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入想取消的订单编号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>360</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="orderIDLineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>220</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
